package/tests/test_server.py

```diff
@@ -106,6 +119,7 @@ def de_pipeline():
                 outputs=["pig", "giraffe"],
                 name="shark",
                 tags=["medium", "large"],
+                namespace="pipeline1.data_engineering",
```

Comment:
Could we add another pipeline called `pipeline2.data_engineering`? I'd like to see how it behaves when there is conflict in names.